File: Components/WinForms/CS/GaugesMainDemo/Modules/ColorSchemeGauges.cs
using DevExpress.XtraEditors;
using DevExpress.XtraGauges.Win;

namespace DevExpress.XtraGauges.Demos {
    public partial class ColorSchemeGauges : TutorialControl {
        public ColorSchemeGauges() {
            InitializeComponent();
            gaugesDashboard1.ColorPicker = colorPickControl;
        }
        void ColorPickControlResultColorChanged(object sender, XtraEditors.InnerColorPickControlSelectedColorChangedEventArgs e) {
            var activeGauge = GetActiveGauge();
            if(activeGauge != null) {
                activeGauge.ColorScheme.Color = e.NewColor;
                gaugesDashboard1.UpdateColor(e.NewColor);
            }
        }
        GaugeControl GetActiveGauge() {
            if(ActiveControl is GaugesDashboard) {
                return (ActiveControl as GaugesDashboard).ActiveGauge;
            }
            return null;
        }
    }
}

File: Components/WinForms/VB/GaugesMainDemo/Modules/ColorSchemeGauges.vb
Imports DevExpress.XtraEditors
Imports DevExpress.XtraGauges.Win

Namespace DevExpress.XtraGauges.Demos
    Partial Public Class ColorSchemeGauges
        Inherits TutorialControl
        Public Sub New()
            InitializeComponent()
            gaugesDashboard1.ColorPicker = colorPickControl
        End Sub
        Private Sub ColorPickControlResultColorChanged(ByVal sender As Object, ByVal e As XtraEditors.InnerColorPickControlSelectedColorChangedEventArgs) Handles colorPickControl.ResultColorChanged
            Dim activeGauge As GaugeControl = GetActiveGauge()
            If activeGauge IsNot Nothing Then
                activeGauge.ColorScheme.Color = e.NewColor
                gaugesDashboard1.UpdateColor(e.NewColor)
            End If
        End Sub
        Private Function GetActiveGauge() As GaugeControl
            If TypeOf ActiveControl Is GaugesDashboard Then
                Return (TryCast(ActiveControl, GaugesDashboard)).ActiveGauge
            End If
            Return Nothing
        End Function
    End Class
End Namespace
